Import Google.Apis.Logging.V2 package.

Deren-Liao opened the pull request:
Hi Ivan,

I created a new branch that based on q1_master.  Somehow I had problem rebase the old branch to q1_master.   Just redone what I did yesterday.  Please take a look.  

Thanks

ivannaranjo commented:
Please retarget the PR to apply on the feature branch for the feature. Which also means that you need to create the feature branch. I suggest you call it q1_logging_errors, that way we know what work is going on in that branch.

Deren-Liao commented:
Hmm, I am still learning git..   I added another commit that I didn't want it to be added to this pull request.  However, GitHub did this automatically.  So I reverted that commit so as to keep this pull request not changed too much.
